modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -816,13 +836,13 @@ private void saveStatesToVault(Set<NodeWithAttributes> newLogicalTopology, long
      * @param removedNodes Nodes that was removed from a topology and should be removed from zones data nodes.
      * @param revision Revision that triggered that event.
      */
-    private void scheduleTimers(
+    private CompletableFuture<Void> scheduleTimers(
```

Comment:
Add returned value in javadoc with description when this future will be completed.